University of Leicester/Leicester210.xml

```diff
@@ -0,0 +1,328 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+   schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="Leicester210" type="mss">
```

Comment:
NO BLANK SPACES are allowed in either names of files, IDs, or names of folders. you must rename the folder in Github to Leicester or to LeicesterUniversity or anything but NOT "University of Leicester". Computers can not read spaces. 
This is very very important.